pyfolio/utils.py

```diff
@@ -17,10 +17,10 @@
 
 import warnings
 
-import empyrical
 import numpy as np
 import pandas as pd
 from IPython.display import display
+from empyrical.utils import default_returns_func
 
```

Comment:
And here we need `from empyrical.utils import get_data_from_yahoo, ...` so that users that have code using pyfolio.utils. something won't break.

Reply:
Ping @jameschristopher, this is a blocker for 0.8.0.